src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeControllerTest.java

```diff
@@ -200,6 +200,12 @@ public void should_successfully_return_all_envelopes_with_processed_status_for_a
             );
 
         blobProcessorTask.processBlobs();
+
+
+        List<Envelope> envelopes = envelopeRepository.findEnvelopesToUpload(10);
```

Comment:
this method name is too confusing `findEnvelopesToUpload` should be something `findEnvelopesToUploadWithFailureCount` not related to this PR but just an observation